### ISSUE TITLE
Add "number of groups per student" to UI

### ISSUE THREAD

**Issue by divilian** [WheezePuppet/CollegeSim#16]
Add to UI, and then set default slider values for "peer drift rate" and "group drift rate" so they're roughly equal. Note from 3/25 email:

```
     For each time step,
         - every one of a student's attributes will get one chance to drift in response to EACH of their groups
         - every one of a student's attributes will get exactly one chance to drift in response to their peers AS A WHOLE.
     In other words, if I'm a student with 43 friends and in 17 groups, then in my sophomore year I will make 17 die rolls for group drift (for each of my attributes), and 1 die roll for peer drift (for each of my attributes).
```

**Comment by divilian** [comment]
Added; see third plot under Groups tab.
